evergreen/config.yml

```diff
@@ -209,8 +201,38 @@ functions:
       content_type: text/text
       display_name: baas server logs
       optional: true
+  - command: s3.put
```

Comment:
The changes around install_baas_output.log also lets us get stack traces from the baas server if the object-store-tests hang because of a problem on the sync server.